File: Unidad 2/Meta 2.1/Python/Ecosistema.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from typing import List

class Ecosistema(object):
	def __init__(self, nombre: str):
		self.nombre = nombre
	
	def setNombre(self, nombre: str):
		self.nombre = nombre

	def getNombre(self) -> str:
		return self.nombre	

	def __str__(self):
		return "Ecosistema{" + "nombre=" + self.nombre + '}'
	

File: Unidad 2/Meta 2.1/Java/Ecosistema.java
public class Ecosistema {
    private String nombre;

    public Ecosistema(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return "Ecosistema{" + "nombre=" + nombre + '}';
    }   
}